ai/writing-assistant.rst

```diff
@@ -0,0 +1,83 @@
+Writing Assistant
+=================
+
+Zammad's writing assistant tools allow agents to apply AI based changes to
+their text before saving or sending it. You can access the settings in Zammad's
+admin section under *AI > Writing Assistant*. Make sure to configure an
+:doc:`AI provider <provider>` before activating the feature. Otherwise, a
+warning message will tell you to do so.
+
+Default Tools
+-------------
+
+Zammad ships some default tools:
+
+- Improve writing
+- Fix spelling and grammar
+- Expand
+- Simplify
+
+These are useful basic tools for all agents. This is why they are not
+customizable and deletable. If you enable the feature by turning the
+**Writing Assistant** toggle on, these default tools are available to all
+agents.
```

Comment:
Not exactly true: *all* writing assistant tools are customizable and deletable. Incl. the default ones, they will be included as part of the DB seeds. But they can be modified or removed later by the admin similarly how all default ticket overviews + triggers + other objects are mutable.

I suggest we drop this block, unless you want to mention they are not limited to any groups by default.